department-of-veterans-affairs/va.gov-cms
# VACMS-3254: Update R&S term page views.

## Description
See #3254

## Testing done
Visual / behat

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/97363492-0e99c500-1879-11eb-8479-99c048778e30.png)
![image](https://user-images.githubusercontent.com/2404547/97363519-15c0d300-1879-11eb-9b1a-2b64ac810afc.png)
![image](https://user-images.githubusercontent.com/2404547/97363571-2a9d6680-1879-11eb-8b17-ffeb738b6968.png)

## QA steps
- As an administrator, go to `admin/config/system/taxonomy-entity-index/reindex`, check both entity types, and rebuild index.
- When indexing is complete, go to R&S page `/taxonomy/term/292`
- Visually verify that the title reads `All articles in: Sign in`
- Visually verify that rss link is not in footer
- Visually verify that page title sort order is a-z
- Visit `/node/6924` and visually verify that there aren't any link check messages related to term page links (3210)
